### ISSUE TITLE
Issues with the agents detection in customgames as observer

### ISSUE THREAD

**Issue by dequedrapha** [deepsidh9/Live-Valorant-Overlay#12]
Hey, 
I tried to use it to create an cool tournament overlay. I created one and tested in normal games. Everything workeds fine.
But not as a Observer...

### Whats maybe the problem
So i think the problem is, that in custom games the masks of dead players are still visible. So the script will return them as still alive

So i hope, u can help me with my problem and again really nice work at all :)

**Comment by deepsidh9** [comment]
Yes that may be the case that the masks, if bright enough, would make the players' status as alive. I'll try thinking of a solution. If you'd like to try, you can have a look at how much the template matching similarity is when the player is alive vs dead and then if a distinguishing threshold can be found. I'll be grateful if you could provide an image of how a dead player place vs an alive player place looks on the top of the screen in the latest patch in the observer mode.

**Comment by dequedrapha** [comment]
Yeah sure :D  (Resolution 1080p [1920x1080] )

### In this example, the skye of the defenders is dead

![image](https://user-images.githubusercontent.com/73545840/163168777-ceea6ee2-355b-4476-99ce-451187f0f885.png)

### and here alive

![image](https://user-images.githubusercontent.com/73545840/163169094-48aa1cee-0165-48e8-9fc1-af89d2e8c739.png)

If you need anything else I can help you with, please contact me 😄

**Comment by dequedrapha** [comment]
I dont wanna annoy u but any solutions yet? 😄

**Comment by deepsidh9** [comment]
I have yet to try out different things to solve this issue. Any ideas to solve this are absolutely welcome.

**Comment by dequedrapha** [comment]
What about trying to find the hole agent pick without manipulate the image to black & white? wont that include the alpha "color"?
